dependencies/build.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo ${BASH_SOURCE[0]}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)